FirebaseStorageSwift/Sources/SwiftAPIExtension.swift

```diff
@@ -35,19 +35,64 @@ private func getResultCallback<T>(
 }
 
 public extension StorageReference {
+  /**
```

Comment:
I suspect you'll get the same Xcode auto-formatting benefit even if you used the more traditional leading-stars form:

```swift
/**
 * Text
 */
```

This is much more customary and aligns with the rest of the code we already have.

Reply:
I started with the leading stars format and that didn't work, but the three-slash doc style does. Converted to that.